tokio-epoll-uring/src/system/slots.rs

```diff
@@ -538,13 +540,14 @@ impl SlotHandle {
             }
         })
         .await;
-        let res: i32;
-        let was_ready_on_first_poll: bool;
-        match inspect_slot_res {
-            InspectSlotResult::AlreadyDone(r) => {
-                res = r;
-                was_ready_on_first_poll = true;
-            }
+        assert!(poll_count >= 1);
+        assert!(
+            !matches!(inspect_slot_res, InspectSlotResult::NeedToWait),
+            "poll_fn closure returns Pending in that case"
+        );
```

Comment:
There's the unreachable already below.

Reply:
This will go away in the next PR https://github.com/neondatabase/tokio-epoll-uring/pull/39